plugins/dependencies/src/test/kotlin/de/fayard/NonRegression.kt

```diff
@@ -16,6 +18,17 @@ val testResources: File = File(".").absoluteFile.resolve("src/test/resources")
 
 class NonRegression: FreeSpec({
 
+    "Generate rule files for dependency groups with a rawRule" {
+        ALL_DEPENDENCIES_NOTATIONS
+        val rulesDir = File(".").absoluteFile.resolve("src/main/resources/refreshVersions-rules")
+        val file = rulesDir.resolve("dependency-groups-alias-rules.txt")
+        val content = AbstractDependencyGroup.ALL_RULES
+            .sorted()
+            .distinct()
+            .joinToString(separator = "\n\n")
+        file.writeText(content)
+    }
+
```

Comment:
@jmfayard Why did you add that?

Reply:
Oh wait, I see that you actually moved it from one file to another. Still wondering why though.

Reply:
I mean, this test is not about non regression. Maybe we should rename this file to something like `BundledDependencyNotationsTests`?

Reply:
@LouisCAD I only moved it from another test because it belongs together